src/main/java/org/venice/beachfront/bfapi/model/Detection.java

```diff
@@ -3,31 +3,66 @@
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
 import javax.persistence.Table;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.vividsolutions.jts.geom.Geometry;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
 
 @Entity
+@IdClass(DetectionPK.class)
 @Table(name = "__beachfront__detection")
 public class Detection {
 	@Id
-	@Column(name = "detection_id")
-	@JsonProperty("detection_id")
-	private String detectionId;
-	@Column(name = "job_id")
+	@JoinColumn(name = "job_id", nullable = false, columnDefinition = "VARCHAR(64)")
+	@OnDelete(action = OnDeleteAction.CASCADE)
+	@ManyToOne(targetEntity = Job.class, optional = false)
 	@JsonProperty("job_id")
-	private String jobId;
-	@Column(name = "user_id")
-	@JsonProperty("user_id")
-	private String userId;
+	private Job job;
+	@Id
+	@Column(name = "feature_id")
+	@JsonProperty("feature_id")
+	private int featureId;
```

Comment:
I'm not entirely sure this should be int or Integer.  Does it make sense for this to be an int vs string?  Not sure what a feature id is.

Reply:
The DB schema declares this as an int, so I think you're fine with leaving it as an int.

Reply:
👍